sqldelight-compiler/src/main/kotlin/com/squareup/sqldelight/core/compiler/integration/TableIntegration.kt

```diff
@@ -32,7 +32,7 @@ internal fun LazyQuery.adapterProperty(): PropertySpec {
 
 private fun LazyQuery.columns() = when (val parentRule = tableName.parent) {
   is SqlCreateTableStmt -> parentRule.columns
-  is SqlAlterTableStmt -> query.columns.map { it.element.parent as SqlDelightColumnDef }
+  is SqlAlterTableStmt, is SqlCreateVirtualTableStmt -> query.columns.map { it.element.parent as SqlDelightColumnDef }
```

Comment:
@AlecStrong here are the two line changes I needed to make to get the dataAdapter bit back into the template. I'm not super sure about these, hopefully you have some insight. It seems a little weird to me that CreateTableStmt and SqlCreateVirtualTableStmt have different interfaces for accessing the columns.

Reply:
i think this is right!